css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
>  Also, these spaces are for targeting input (colors, images, videos, etc) and not displays.

Makes sense to me... so "image-p3" is a color space that will be used to encode pixels in photos? I do not expect it to be used for professional content. Perhaps this was all discussed before and I appreciate your patience.